server/configs/application.properties

```diff
@@ -14,41 +14,39 @@ server.port=@@serverPort@@
 # HTTP-only port for servers that need to handle both HTTPS (configure via server.port and server.ssl above) and HTTP
 #context.httpPort=8080
 
-context.dataSourceName[0]=jdbc/labkeyDataSource
-context.driverClassName[0]=@@jdbcDriverClassName@@
-context.url[0]=@@jdbcURL@@
-context.username[0]=@@jdbcUser@@
-context.password[0]=@@jdbcPassword@@
-context.maxTotal[0]=50
-context.maxIdle[0]=10
-context.maxWaitMillis[0]=120000
-context.accessToUnderlyingConnectionAllowed[0]=true
-context.validationQuery[0]=SELECT 1
-
-#context.logQueries[0]=true
-#context.displayName[0]=Alternate Display Name
-
-#context.dataSourceName[1]=jdbc/@@extraJdbcDataSource@@
```

Comment:
Does the new style not still need a way to specify alternate data sources? You have the other 'extra' vars accounted for. below.

Reply:
Yes, it does. I've added a comment to try to clarify how this works. Let me know if it helps or not.

Reply:
ah, yeah, that makes sense. thanks.